lib/active_merchant/billing/gateways/cyber_source_rest.rb

```diff
@@ -77,6 +77,26 @@ def verify(credit_card, options = {})
         end
       end
 
+      def store(payment, options = {})
+        MultiResponse.run do |r|
+          if options[:customer_id].nil?
+            customer_id = r.process { commit('customers', create_customer_request(payment, options)) }.params['id']
+            break r unless r.success?
```

Comment:
I think `break` is usually used to escape a block without returning a value, but we're returning something here, so it should probably be `return`?